Gopkg.toml

```diff
@@ -180,7 +180,11 @@
 
 [[constraint]]
   name = "github.com/go-acme/lego"
-  version = "2.6.0"
+  version = "2.7.1"
+
+[[constraint]]
```

Comment:
What is the reason for adding the protobuf dependency here?

Reply:
It won't build anymore because of a sub-dependency. Without protobuf < 1.3 you'll get:
```
---> Making bundle: binary (in .)
# github.com/containous/traefik/vendor/github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1
vendor/github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1/common.pb.go:22:11: undefined: proto.ProtoPackageIsVersion3
# github.com/containous/traefik/vendor/github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1
vendor/github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1/resource.pb.go:21:11: undefined: proto.ProtoPackageIsVersion3
```